packages/contracts-bedrock/src/libraries/errors/CommonErrors.sol

```diff
@@ -12,3 +12,6 @@ error NotCustomGasToken();
 
 /// @notice Error for when a transfer via call fails.
 error TransferFailed();
+
+/// @notice Thrown when attempting to perform an operation and the account is the zero address.
+error ZeroAddress();
```

Comment:
Since it is only used on `SuperchainERC20`, not sure it needs to go here. Wdyt?

Reply:
it's used on OptimismSuperchainERC20 as well

Reply:
Yes, got blind 🙏